docs/src/pages/components/autocomplete/GitHubLabel.js

```diff
@@ -1,110 +1,28 @@
 /* eslint-disable @typescript-eslint/no-use-before-define */
 import * as React from 'react';
 import PropTypes from 'prop-types';
-import { useTheme, makeStyles } from '@material-ui/core/styles';
+import { useTheme, experimentalStyled as styled } from '@material-ui/core/styles';
```

Comment:
@vicasas @oliviertassinari would be great to have another one look into the changes on this demo. It was a bit challenging